### ISSUE TITLE
SQL Ip never changed from 172.18.0.10

### ISSUE THREAD

**Issue by Tydium** [GeyserMC/GeyserDiscordBot#153]
Im trying to setup my sql to the ip 172.17.0.1 so it can actually connect however I've having trouble in the first place since it wants to always use the other ip.

**Comment by Chew** [comment]
Not sure how:
https://github.com/GeyserMC/GeyserDiscordBot/blob/f80fdd1b8d5c6fbcf6669c760489fb56a9daef11/src/main/java/org/geysermc/discordbot/storage/MySQLStorageManager.java#L46

The specified IP never appears in the code...

**Comment by Tydium** [comment]
That is very strange, I tried looking there also but couldn't find anything. My bot.prop file has 172.17.0.1 so I'm sorta just confused.

**Comment by Tydium** [comment]
Since I'm using this on Pterodactyl it was kinda weird that this wasn't working, however RTM gave me the idea for the fix. He said that the 172.18.0.10 number was the docker IP not the ip im trying to connect to. So basically what I did was is was I deleted my old database then made a new one, but before I created it I made sure to allow connections only from that IP address, and it seemed to fix the problem.
